### PR TITLE
reformat absolute quant

### PR DESCRIPTION
**Pull request by ypriverol**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/fdr.rst [hunk 1]
```diff
@@ -28,31 +28,32 @@ PSM or peptide level
...
 The first FDR control happens after PSM re-scoring by either Percolator or a distribution-fitting approach.
-Percolator outputs its own PSM- or peptide-level FDR with correction for an estimated rate of incorrect targets (TODO cite),
+Percolator outputs its own PSM- or peptide-level FDR with correction for an estimated rate of incorrect targets,
 therefore this usually improved estimate is used when Percolator was chosen.
 
 .. warning:: Choosing peptide-level FDR with Percolator will discard all but the best PSM per peptide. Use with caution
     if you need full traceability.
 
-When using OpenMS' distribution-fitting approach, a standard formula for FDR calculation is used
-(TODO add formula) and is currently only available for the PSM level.
```
Comment (jpfeuffer) [inline docs/fdr.rst:38]: I think this is SUPER important. A lot of people want to know how FDR control is done to compare with other tools!

File: docs/fdr.rst [hunk 2]
```diff
@@ -28,31 +28,32 @@ PSM or peptide level
...
-
 The FDR filtering at peptide spectrum match (PSM) level is currently always applied at the single file level.
 We argue that experiment-wide FDR control at the end of the workflow on the protein level is sufficient to limit error
 rates for the overall analysis. Nonetheless, an option for experiment-wide re-scoring and FDR control on PSM/peptide-level
 is under consideration.
 
 .. hint:: The chosen FDR cutoff for PSMs/peptides influences the amount of PSMs available for ID-based feature
-    finding, feature linking, as well as protein inference and grouping (TODO internal links).
+    finding, feature linking, as well as protein inference and grouping (see protein :doc:`inference`).
```
Comment (jpfeuffer) [inline docs/fdr.rst:43]: I meant links to all steps

File: docs/identification.rst [hunk 3]
```diff
@@ -1,20 +1,10 @@
...
 
-.. toctree::
-   :maxdepth: 1
-
-   searchengines
-   consensusid
-   rescoring
-   fdr
-   modlocal
-
```
Comment (jpfeuffer) [inline docs/identification.rst:12]: If the toc tree is under a subsection it won't work correctly. It has to be at the start.

File: docs/lfq.rst [hunk 4]
```diff
@@ -1,26 +1,81 @@
...
+proteomicsLFQ supports **ID-based feature extraction** or a combined **ID-based + untargeted extraction**.
+
+1. **ID-based feature extraction** uses targeted feature detection using RT and m/z information derived from identification data to extract features. Only identifications found in a particular MS run are used to extract features in the same run. No transfer of IDs (match between runs) is performed.
+2. **ID-based + untargeted extraction** adds untargeted feature detection to obtain quantities from unidentified features. Transfer of IDs (match between runs) is performed by transferring feature identifications to coeluting, unidentified features with similar mass and RT in other runs.
+
+
+To enable MBRs, the users should enable the following parameter: `--targeted_only false`
+
+.. note:: Match between runs is performed by transfering feature identifications to coeluting, unidentified features with similar mass and RT in other runs.
+   Feature extra
```
Comment (jpfeuffer) [inline docs/lfq.rst:49]: ?

File: docs/lfq.rst [hunk 5]
```diff
@@ -1,26 +1,81 @@
...
+
+**Peptides used for quantification**: By default, protein quantification is performed only in unique peptides
+`protein_quant='unique_peptides'`, if the users prefer to include also use shared peptides `protein_quant 'shared_peptides'`.
+
+**Quantification Method**: LFQ analytical methods uses two main approaches for quantification intensity-based and
+spectral count algorithms. By default, proteomicsLFQ uses the most reliable method intensity-based
+`quantification_method = 'feature_intensity'`, however if the users wants to quantified based on spectral counting, the
+can pass the following option `quantification_method 'spectral_counting'`.
+
+**Quantify Decoys**: By default, the quantms do not quantified decoy peptide hits, this behaviour can be changed by using the paramter `quantify_decoys`
```
Comment (jpfeuffer) [inline docs/lfq.rst:33]: Auto enabled for triqler!

File: docs/lfq.rst [hunk 6]
```diff
@@ -1,26 +1,81 @@
 Label-free quantification with DDA (LFQ)
 ========================================
 
-.. toctree::
```
Comment (jpfeuffer) [inline docs/lfq.rst:4]: Same here. Toc won't work correctly

File: docs/lfq.rst [hunk 7]
```diff
@@ -1,26 +1,81 @@
...
+
+proteomicsLFQ supports **ID-based feature extraction** or a combined **ID-based + untargeted extraction**.
+
+1. **ID-based feature extraction** uses targeted feature detection using RT and m/z information derived from identification data to extract features. Only identifications found in a particular MS run are used to extract features in the same run. No transfer of IDs (match between runs) is performed.
+2. **ID-based + untargeted extraction** adds untargeted feature detection to obtain quantities from unidentified features. Transfer of IDs (match between runs) is performed by transferring feature identifications to coeluting, unidentified features with similar mass and RT in other runs.
+
+
+To enable MBRs, the users should enable the following parameter: `--targeted_only false`
+
+.. note:: Match between runs is performed by transfering feature identifications to coeluting, unidentified features with similar mass and RT in other runs.
```
Comment (jpfeuffer) [inline docs/lfq.rst:48]: I think this is wrong. You should have left it like before.
You are talking about MBR multiple times with different concepts.

File: docs/triqler.rst [hunk 8]
```diff
@@ -1,2 +1,41 @@
 Triqler
 =======
+
+`Triqler <https://github.com/statisticalbiotechnology/triqler>`_ is a probabilistic graphical model that propagates error
+information through all steps from MS1 feature to protein level, employing distributions in favor of point estimates,
+most notably for missing value imputation [THE2018]_. The model outputs posterior probabilities for fold changes between treatment
+groups, highlighting uncertainty rather than hiding it.
```
Comment (jpfeuffer) [inline docs/triqler.rst:7]: Mention it only works for LFQ and automatically activates decoy quantification which makes the pipeline a little slower.

**Review by jpfeuffer — COMMENTED** [review]
See my comments

File: docs/lfq.rst (hunk 7 above)
Comment (ypriverol) [reply docs/lfq.rst]: @jpfeuffer can you help me with this part. I want to explain the MBR and how to run it in the pipeline. The previous discussion is more focus in multiple concepts of openMS.

**Pull request merged**